test_rigs/unit-tests/configs/cisco_enable

```diff
@@ -6,5 +6,6 @@ enable secret 5 $1$aaaaaaaaaaaaaaaaaaaaaaaaaaa
 enable super-user-password aaaaaaaaaaaaaaaaaaaaa
 enable telnet authentication
 enable telnet password aaaaaaaaaaaaaaaaaaaaaaaaa
+enable password $sha512$5000$98pqLCa2lpNvN6gnQl3zyQ==$QvdeF/4EqAATT2/e64ujg== woldj4
```

Comment:
What configuration language is this? I cannot enter this on my ASA VM. Also, it looks like it contains an invalid base64 string after the last `$`.

Reply:
The password was from an ASA and 'looks like' that (the important part that breaks parsing are the random `$` symbols and the last part) -- but in reality I just randomly changed various characters so it wouldn't match, so I'm not surprised it's not a valid base64 string.

I didn't create the ASA config, @asydney did, so if the password isn't sensitive I suspect he could share the original string if you need it.

Reply:
No I just made it valid base64 by inserting a character. It's just a hash, so I don't need anything original.